checker/checker_not_breaking_test.go

```diff
@@ -270,9 +279,17 @@ func TestBreaking_TagAddedWithCustomCheck(t *testing.T) {
 	s2.Spec.Paths[securityScorePath].Get.Tags = append(s2.Spec.Paths[securityScorePath].Get.Tags, "newTag")
 	d, osm, err := diff.GetWithOperationsSourcesMap(getConfig(), &s1, &s2)
 	require.NoError(t, err)
-	errs := checker.CheckBackwardCompatibility(checker.GetChecks(utils.StringList{"api-tag-removed"}), d, osm)
-	for _, err := range errs {
-		require.Equal(t, checker.ERR, err.Level)
-	}
-	require.Empty(t, errs)
+	verifyNonBreakingChangeIsChangelogEntry(t, d, osm, "api-tag-added")
+}
+
+// BC: adding an operation ID is not breaking with "api-operation-id-removed" check
```

Comment:
Please update the omment to CL

Reply:
Updated, but let's discuss further at https://github.com/Tufin/oasdiff/pull/297/commits/9b30cba92c6e99abceed88dfe536e0830ed7620b#r1233864561